lms/views/api/lti.py

```diff
@@ -115,6 +115,8 @@ def get_speedgrader_launch_url(self):
             "focused_user": parsed_params["h_username"],
             "learner_canvas_user_id": parsed_params["learner_canvas_user_id"],
             "group_set": parsed_params.get("group_set"),
+            "ext_lti_assignment_id": parsed_params.get("ext_lti_assignment_id"),
```

Comment:
What is this one for?

Reply:
Both `ext_lti_assignment_id` and `resource_link_id` identify an assignment on canvas.